packages/blitz/src/cli/utils/codegen-tasks.ts

```diff
@@ -37,15 +38,33 @@ export const codegenTasks = async () => {
       (name) => name === "prisma",
     )
 
+    const tempDir = os.tmpdir() 
+    const creationDate = await fs.stat(process.cwd())
+    if (!fs.existsSync(join(tempDir, `blitz-${creationDate.birthtimeMs}-prisma-client.json`))) {
+      const lastModified = await fs.stat(join(process.cwd(), "db/schema.prisma"))
+      await fs.writeFile(
+        join(tempDir, `blitz-${creationDate.birthtimeMs}-prisma-client.json`),
+        JSON.stringify({lastModified: lastModified.mtimeMs}),
+      )
+    }
```

Comment:
I don't mean to be picky, but I think we should only be doing this if `hasPrisma` is true.

Reply:
@dillonraphael I have made the requested change